frontend/components/Modal/_styles.scss

```diff
@@ -49,6 +49,7 @@
     border-bottom: 1px solid $ui-fleet-black-10;
     display: flex;
     justify-content: space-between;
+    min-height: 37px;
```

Comment:
prevents jumping of header on include/exclude `x` due to height of child element